pkg/devspace/build/create_builder.go

```diff
@@ -19,14 +19,19 @@ import (
 // createBuilder creates a new builder
 func (c *controller) createBuilder(imageConfigName string, imageConf *latest.ImageConfig, imageTag string, options *Options, log log.Logger) (builder.Interface, error) {
 	var err error
-	var builder builder.Interface
+
+	if c.builder != nil {
```

Comment:
This cannot stay, this change caused a severe issue in an older version (#949), because 2 or more images are always build with the same builder.

Reply:
Fixed in commit de974eb355301581d1f5d4158678d0a0d3876a83